### ISSUE TITLE
1/8th Hinges, Rotators, Pistons

### ISSUE THREAD

**Issue by BigBadKangaroo** [tsunamayo/Starship-EVO#5813]
Hi Tsuna,

I know that this has been a thing that a lot of player have wanted for a long time, and I know that you never were enthusiastic about it, but please just look at what a 1/8 Piston, Hinge, Rotator for details is capable of: 
(I'm Sorry TwoNiner for using your build as a refference, but it's the perfect example piece for that)
https://steamcommunity.com/sharedfiles/filedetails/?id=3268960578

![20240616152755_1](https://github.com/tsunamayo/Starship-EVO/assets/81495956/2169d4ff-28da-410f-ae0e-6ae7ee064b00)
![20240616152831_1](https://github.com/tsunamayo/Starship-EVO/assets/81495956/7e80d2bb-89f2-4c12-b6cf-e5e14fdd1519)
![20240616155314_1](https://github.com/tsunamayo/Starship-EVO/assets/81495956/6c9ee442-d275-4cd2-9588-0984be90983c)
![20240616155342_1](https://github.com/tsunamayo/Starship-EVO/assets/81495956/483d5c0a-714f-4884-9b7e-1ec0c4e175a4)

I want to build the small "eye patches" for my AT-ST on a hinge so they can be opend and closed, I either have to use a Rotator in a strange place (which I honestly dislike), or I have to use the really big 1/4th hinges that just break the design:
![20240616152905_1](https://github.com/tsunamayo/Starship-EVO/assets/81495956/72ff4734-d5dc-484b-bdd6-1cc19a4d9a24)

I know that this is a niche thing and mainly only for the really detail-loving builders, but please make this happen.
This has been an on-going thing but since Mechs will be a real thing in the near future, this topic becomes relevant again!

Not every Mech has to be 10,20,30 meters tall ... I'm thinking of personal armor suits not much bigger than the pilot? Buildable Armor pieces in general? Or maybe even robotic AI crewmember that we can actually build? (maybe one day) 

Just please make this happen. Even if this is just a "details only" thing, It would be a really awesome addition for buildng!

**Comment by ProPeach** [comment]
And of course, 1/8th Mech Joints too!

I really don't think that 1/8th mechanisms are a niche desire, I've actually seen quite a few new players posting on the Discord thinking they're doing something wrong because they can't find a 1/8th hinge for their 1/8th brick shape.

It's still the most popular suggestion in the Discussions forum #3955. Could you talk a little about your reasons for not not wanting them Tsuna? It could help us understand and talk about it, from an outside perspective I find it tough to think of a reason *not* to enable the smaller scaling lol as it seemed to function well from those tests that Nuclear Lemons did a while ago.

**Comment by ZachZent** [comment]
![image](https://github.com/tsunamayo/Starship-EVO/assets/36980446/6b938176-eb72-47aa-afe7-df4b9c54a154)
(from Vor)
It's wacky

**Comment by Gallt** [comment]
yeah, i run into situations quite often i'd like some small mechanisms.

**Comment by king32109** [comment]
have seen situations where i really could have used a 1/8 hinge or piston but coudlnt as they werent that small
